unattended_installer/install_functions/opendistro/elasticsearch.sh

```diff
@@ -169,6 +170,43 @@ function installElasticsearch() {
 
 }
 
+function uninstallelasticsearch() {
+    logger "Elasticsearch will be uninstalled."
```

Comment:
Use a blank line separator from the first function line to function definition

Reply:
Done https://github.com/wazuh/wazuh-packages/pull/1211/commits/c3fc83525f9efa17705f53a3ff6e1f8d681d1d49